packages/unit-testing/stubs/search/Result.js

```diff
@@ -1,4 +1,4 @@
-define([], function () {
+define(['./Column'], function (Column) {
```

Comment:
- recordType should be search.Type type
- id type is number in code and string in documentation
- options.summary type in getValue(options) method is Summary in code and string in documentation
- options.summary type in getText(options) method is Summary in code and string in documentation
- opttions.func parameter is not documented neither in getText or getValue

Reply:
Ignoring Search module